db/key/keyModel.js

```diff
@@ -0,0 +1,15 @@
+const db = require('./../index.js').db;
```

Comment:
On line 1, you don't need to write `./` if you are going to follow it with a `../`.